web-client/src/lib.rs

```diff
@@ -343,6 +346,43 @@ impl WebClient {
 
         Ok(())
     }
+
+    /// This function is used to query the network for transactions from some specific adress, that
+    /// have been included in the chain.
+    /// The obtained transactions correspond to extended transactions.
+    /// They are verified before being returned.
+    #[wasm_bindgen(js_name = requestTransactionsFromAddress)]
+    pub async fn request_transations_from_address(
+        &self,
+        address: Address,
+        max: u16,
+    ) -> Result<Uint8Array, JsError> {
+        let transactions = self
+            .inner
+            .consensus_proxy()
+            .request_transactions_by_address(address.get_native(), Some(max))
+            .await?;
+
+        let executed_txs: Vec<ExecutedTransaction> = transactions
+            .into_iter()
+            .map(|ext_transaction| ext_transaction.into_transaction().unwrap())
+            .collect();
+
+        // TODO: We are converting executed transactions into a regular transaction, which loses the execution result of the txn
```

Comment:
I could imagine adding a property to the JS transaction that has the execution result, then it can be accessed from JS on the struct.

Reply:
Just remembered that the 1.0 client had a separate entity `TransactionDetails`  that had additional properties to the tx data, like status, block number, timestamp. I'll add it after this is merged.

Reply:
Code prepared and ready to go when this is merged: https://github.com/nimiq/core-rs-albatross/compare/viquezcl/wasm-infra...soeren/wasm-plain-transaction?expand=1